assets/blocks/course-outline/status-preview/status-store.js

```diff
@@ -311,9 +314,7 @@ const reducers = {
 	DEFAULT: ( action, state ) => state,
 };
 
-export const COURSE_STATUS_STORE = 'sensei/course-status';
-
-registerStore( COURSE_STATUS_STORE, {
+export const COURSE_STATUS_STORE = createStore( 'sensei/course-status', {
```

Comment:
The export changed here from the store name to the store descriptor. I checked that it's only used for the selects and dispatches, and the name is the old way to use them.

So this change also fixes this.